.github/workflows/release-build.yml

```diff
@@ -84,6 +88,11 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
+    - if: inputs.publish
+      env:
+        SIGNING_KEY: ${{ secrets.signingkey }}
```

Comment:
We also need to change our just-setup to always install cargo-binstall explicitly

Reply:
How about using our own action instead

Reply:
That would be great!